evadb/udfs/chatgpt.py

```diff
@@ -104,8 +107,12 @@ def forward(self, text_df):
                     }
                 ],
             }
+            start = time.time()
             response = openai.ChatCompletion.create(**params)
             results.append(response.choices[0].message.content)
+            diff = 20 - time.time() + start
```

Comment:
Please add a comment saying we will fix this later (like to do support rate limit).

Reply:
We can borrow similar logic from Langchian. See #829 for reference.
```        
callback_manager = CallbackManager([StreamingStdOutCallbackHandler()])
self.llm = GPT4All(
    model=self.model_path,
    backend="gptj",
    callbacks=callback_manager,
    verbose=False,
)
```

Reply:
@affan00733